source/stylesheets/card.scss

```diff
@@ -6,7 +6,30 @@
   border-radius: $base-border-radius;
   height: $card-height;
   margin: $base-vh-spacing auto 0;
+  transition: $base-transition;
+  position: absolute;
+  left: 50%;
+  margin-left: -$card-width / 1.77;
+  top: 7.75vh;
+  z-index: -100;
+ }
+
+.card--transform {
+  transform: rotateY(180deg);
+}
+
+.card--layers {
+  background-color: $red;
+  border-radius: $base-border-radius;
   box-shadow: $bkg-card-fill;
+  height: $card-height;
+  padding: 1em;
+  position: absolute;
+  width: $card-width;
+  z-index: -200;
+  left: 50%;
+  margin-left: -$card-width / 1.9;
```

Comment:
Same comments regarding z-index, margin, and positioning.

Since these z-indexes are related, they could be `$card-z-index`, `$card-z-index - 1`, and `$card-z-index - 2`, for example.

Reply:
Added z-index variables in the latest commit + tried to apply more variables for spacing.